www/common/outer/async-store.js

```diff
@@ -1220,7 +1221,20 @@ define([
                 if (!s.id) { inMyDrive = res.length; }
 
                 Array.prototype.push.apply(allData, res);
+
+                if (res && res[0]) {
+                    var elementPath = store.manager.user.userObject.findFile(res[0].id)[0];
```

Comment:
Multiple issues here:
1. "res" may reference data from a team drive but you're trying to search the path from your own drive
2. "res" is an array because the pad may be stored in your drive multiple times. You're pushing to the array only the path of the first occurence found